src/plots/barPlot.ts

```diff
@@ -500,7 +500,7 @@ export class Bar<X, Y> extends XYPlot<X, Y> {
 
       let barWidth = attrToProjector["width"](d, i, dataset);
       let barHeight = attrToProjector["height"](d, i, dataset);
-      let text = this._labelFormatter(valueAccessor(d, i, dataset));
+      let text = this._labelFormatter(valueAccessor(d, i, dataset), d, [dataset]);
```

Comment:
What are the semantics of the last argument now? barPlot allows multiple datasets and will draw all of them; this allows you to e.g. color bars by dataset. IMO we could pass the whole dataset array, and/or just the dataset associated with this datum, but passing an array holding just the single dataset is confusing to me